dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/RequestItemRepository.java

```diff
@@ -229,14 +231,20 @@ public RequestItemRest put(Context context, HttpServletRequest request,
         }
 
         // Check for authorized user
-        RequestItemAuthor authorizer;
+        List<RequestItemAuthor> authorizers;
         try {
-            authorizer = requestItemAuthorExtractor.getRequestItemAuthor(context, ri.getItem());
+            authorizers = requestItemAuthorExtractor.getRequestItemAuthor(context, ri.getItem());
         } catch (SQLException ex) {
             LOG.warn("Failed to find an authorizer:  {}", ex.getMessage());
-            authorizer = new RequestItemAuthor("", "");
+            authorizers = Collections.EMPTY_LIST;
         }
-        if (!authorizer.getEmail().equals(context.getCurrentUser().getEmail())) {
+
+        boolean authorized = false;
+        String requester = context.getCurrentUser().getEmail();
+        for (RequestItemAuthor authorizer : authorizers) {
+            authorized |= authorizer.getEmail().equals(requester);
+        }
+        if (!authorized) {
```

Comment:
this is not related to this PR as the behavior is not changing but I want to raise a question that can be eventually moved to a followup ticket if other feel reasonable. Why are we checking authorization here? the user has a valid token, maybe the request email as been forwarded to a trusted colleague asking to proceed in this way we will prevent that. This also require the user to be loggedin and maybe the author is external or not longer affiliated with the repo (this would require also removing the auth annotation check on the method)

Reply:
just discovered that this is the topic of https://github.com/DSpace/DSpace/pull/8319